Source/WebCore/platform/graphics/mac/controls/InnerSpinButtonMac.mm

```diff
@@ -90,8 +89,7 @@
         context.scale(style.zoomFactor);
     }
 
-    LocalCurrentGraphicsContext localContext(context);
```

Comment:
Was this just always unnecessary?

Reply:
I think yes. `[NSAppearance _drawInRect:]` is just a wrapper for a CoreUI method which takes a `CGContextRef` and does not rely on `[NSGraphicsContext currentContext]`.